src/poker.js

```diff
@@ -1,5 +1,43 @@
+const cardValue = { 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, J: 11, Q: 12, K: 13, A: 14 }
+
 class Poker {
-  // Write your implementation here
+  isPair (hand) { return hand[0] === hand[1] || hand[0] === hand[2] || hand[1] === hand[2] }
+
+  isThreeOfAKind (hand) { return hand[0] === hand[1] && hand[1] === hand[2] }
```

Comment:
Already evaluating if there is a pair /three of a kind with these methods... both one-line code, super neat!